src/OpenTelemetry.Resources.Process/ProcessDetector.cs

```diff
@@ -14,12 +14,25 @@ internal sealed class ProcessDetector : IResourceDetector
     /// <returns>Resource with key-value pairs of resource attributes.</returns>
     public Resource Detect()
     {
+        var currentProcess = System.Diagnostics.Process.GetCurrentProcess();
+#pragma warning disable CA1305 // Specify IFormatProvider
```

Comment:
These CA13xx warnings should be fixed by using the invariant culture, not suppressed.